docs/excel/custom-functions-debugging.md

```diff
@@ -54,18 +54,18 @@ You can use the browser developer tools to debug custom functions that don't use
 
 ### Start debugging
 
-1. Open developer tools in the browser. For Chrome and most browsers F12 will open the developer tools.
+1. Open developer tools in the browser. For Chrome and most browsers F12 opens the developer tools.
 1. In developer tools, open your source code script file using <kbd>Cmd</kbd>+<kbd>P</kbd> or <kbd>Ctrl</kbd>+<kbd>P</kbd> (**functions.js** or **functions.ts**).
 1. [Set a breakpoint](https://code.visualstudio.com/Docs/editor/debugging#_breakpoints) in the custom function source code.
 
-If you need to change the code you can make edits in VS Code and save the changes. Refresh the browser to see the changes loaded.
+Edit code in VS Code, save, then refresh the workbook page to load updates.
 
 ## Use the command line tools to debug
 
-If you aren't using VS Code, you can use the command line (such as bash, or PowerShell) to run your add-in. You'll need to use the browser developer tools to debug your code in Excel on the web. You cannot debug the desktop version of Excel using the command line.
+If you aren't using VS Code, you can use the command line such as bash or PowerShell to run your add-in. Use the browser developer tools to debug your code in Excel on the web. You cannot debug the desktop version of Excel from the command line.
```

Comment:
I'm not sure the original statement that you "can't debug desktop Excel from the command line" is true. You can debug by right-clicking in the task pane and choosing debug to open the developer tools.

Reply:
Do you think it would be sufficient to simply remove the sentence?
```suggestion
If you aren't using VS Code, you can use the command line such as bash or PowerShell to run your add-in. Use the browser developer tools to debug your code in Excel on the web.
```